### PR TITLE
Fix React Native sample highlighting

### PR DESCRIPTION
**Pull request by krollins-mdb**
Due to code format changes, highlighting was incorrect in a large number of samples. Eventually, I want to migrate all of the manual highlighting to use Bluehawk :emphasize: tags, but that's beyond the scope of this simple fix-it PR.

Staged changes:
- [Manage Sync Subscriptions](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/sync-data/flexible-sync/)
- [Dictionaries](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/model-data/data-types/dictionaries/)
- [Mixed](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/model-data/data-types/mixed/)
- [Sets](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/model-data/data-types/sets/)
- [Embedded Objects](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/model-data/data-types/embedded-objects/)
- [Create](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/crud/create/)
- [Delete](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/crud/delete/)
- [Query Data](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/crud/query-data/)
- [Read](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/crud/read/)
- [Define a Realm Object Model](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/model-data/define-a-realm-object-model/)
- [Quick Start](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/quick-start/)

### CODE REVIEW

File: source/sdk/react-native/crud/create.txt [hunk 1]
```diff
@@ -112,7 +112,7 @@ To create an object with a to-many relationship to another object:
 
 .. literalinclude:: /examples/generated/react-native/ts/create-test.test.snippet.crud-create-to-many-object.tsx
    :language: typescript
-   :emphasize-lines: 2, 8-14, 21
+   :emphasize-lines: 2, 8-14, 20
```
Comment (cbullinger) [inline source/sdk/react-native/crud/create.txt:115]:
```suggestion
   :emphasize-lines: 2, 8-14, 25
```
should be highlighting the `onPress={() => handleCreateCompany()}` line?

File: source/sdk/react-native/crud/delete.txt [hunk 2]
```diff
@@ -121,14 +121,12 @@ In the following example of a ``DogList`` component, we:
       
       .. literalinclude:: /examples/generated/react-native/ts/delete.test.snippet.crud-delete-multiple-objects.tsx
          :language: typescript
-         :emphasize-lines: 3, 6-9, 12-14
```
Comment (cbullinger) [inline source/sdk/react-native/crud/delete.txt:124]: was there a reason you removed highlighting from only this section's examples?

File: source/sdk/react-native/crud/read.txt [hunk 3]
```diff
@@ -112,14 +112,14 @@ In the following example of a ``TaskList`` component, we:
       
       .. literalinclude:: /examples/generated/react-native/ts/read.test.snippet.crud-read-filter-data.tsx
          :language: typescript
-         :emphasize-lines: 3, 6, 9-11
+         :emphasize-lines: 4-10, 12-18
```
Comment (cbullinger) [inline source/sdk/react-native/crud/read.txt:115]:
```suggestion
         :emphasize-lines: 4-10, 13-18
```
so the second highlighted block doesn't include the comment (for consistency with the first)

File: source/sdk/react-native/crud/read.txt [hunk 4]
```diff
@@ -112,14 +112,14 @@ In the following example of a ``TaskList`` component, we:
...
 
    .. tab::
       :tabid: javascript
 
       .. literalinclude:: /examples/generated/react-native/js/read.test.snippet.crud-read-filter-data.jsx
          :language: javascript
-         :emphasize-lines: 3, 6, 9-13
+         :emphasize-lines: 4-10, 12-18
```
Comment (cbullinger) [inline source/sdk/react-native/crud/read.txt:122]:
```suggestion
         :emphasize-lines: 4-10, 13-18
```

File: source/sdk/react-native/quick-start.txt [hunk 5]
```diff
@@ -149,14 +149,14 @@ To learn more, refer to the :ref:`Query Data <react-native-query-data>`.
    
       .. literalinclude:: /examples/generated/react-native/ts/quickstart-find-sort-filter.test.snippet.find-sort-filter.tsx
          :language: typescript
-         :emphasize-lines: 44, 45
+         :emphasize-lines: 46, 56
```
Comment (cbullinger) [inline source/sdk/react-native/quick-start.txt:152]: should this be highlighting the entire range? [current example ](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/fix-rn-highlighting/sdk/react-native/quick-start/#find--sort--and-filter-objects) doesn't look right
```suggestion
         :emphasize-lines: 46-56
```

File: source/sdk/react-native/quick-start.txt [hunk 6]
```diff
@@ -149,14 +149,14 @@ To learn more, refer to the :ref:`Query Data <react-native-query-data>`.
...
    
    .. tab::
       :tabid: javascript
 
       .. literalinclude:: /examples/generated/react-native/js/quickstart-find-sort-filter.test.snippet.find-sort-filter.jsx
          :language: javascript
-         :emphasize-lines: 37, 38
+         :emphasize-lines: 40, 50
```
Comment (cbullinger) [inline source/sdk/react-native/quick-start.txt:159]:
```suggestion
         :emphasize-lines: 40-50
```

**Review by cbullinger — APPROVED** [review]
A couple Qs but overall LGTM

File: source/sdk/react-native/crud/delete.txt (hunk 2 above)
Comment (krollins-mdb) [reply source/sdk/react-native/crud/delete.txt:124]: I should have left a comment about these. For this one and a couple other examples, I felt like there was just too much highlighting. And when everything is important, nothing is important. ha. It's more a problem with the samples though - they're trying to do too many things.

File: source/sdk/react-native/quick-start.txt (hunk 5 above)
Comment (krollins-mdb) [reply source/sdk/react-native/quick-start.txt:152]: It's definitely not right. I'll fix this one locally and push up the change.

**Pull request merged**